waternoose/src/main/java/com/hartwig/healthchecks/waternoose/check/MetadataChecker.java

```diff
@@ -36,6 +36,7 @@
     private static final String LOG_FILENAME_FORMAT = "%s.log";
     private static final String DATE_OUT_FORMAT = "yyyy-MM-dd";
     private static final String DATE_IN_FORMAT = "[EEE MMM d HH:mm:ss z yyyy]"
+                                                 + "[EEE MMM  d HH:mm:ss z yyyy]"
```

Comment:
Hard-coding the space doesn't seem right when `dd` is already supposed to be the padded variant. Does `dd` in place of ` d` fix your locale?